RELEASE-NOTES.md

```diff
@@ -7,7 +7,7 @@ The format of this file is based on [Keep a Changelog](http://keepachangelog.com
 
 The Axom project release numbers follow [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - Release date yyyy-mm-dd
+## [Version 0.4.0] - Release date 2020-09-22
```

Comment:
i can never remember when unreleased gets added back? is it the first person to do a change or this PR?

Reply:
good call, I guess it's the first person?

Reply:
Also, should this be `2020-09-23` to match the date on the release page?

Reply:
That's what we have been doing -- the first person that merges, adds the `Unreleased` section.

I was thinking about that yesterday. Another, perhaps smoother way to do this would be to branch from `main` (or keep the release candidate branch for that around), make a commit that adds the "Unreleased" section  and merge that into develop

Reply:
In BLT, i think ive just left a blank unreleased section at top.

Reply:
Having a blank unreleased section at the top seems a bit odd to me. 

The more I think about this, I think I like the idea of:

1.  Keeping the "v0.4.0-rc" branch around it gets merged back into main.
2. Tag `main`  
3. Merge `main` back into the "v0.4.0-rc" branch (this will effectively just bring in the merge-commit and the tag)
4. Add a commit with the blank unreleased section at the top
5. Marge the "v0.4.0-rc" back to develop

Maybe this can be discussed in a future meeting...

Reply:
Just pick something and go with it =)